lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -710,6 +710,7 @@ class MailboxDashBoardController extends ReloadableController
     if (PlatformInfo.isWeb) {
       injectWebSocket(session, currentAccountId);
     } else {
+      injectWebSocket(session, currentAccountId);
```

Comment:
Shouldn't websocket only be used when in foreground?

Reply:
yes, for sure. This controller run only when app in foreground.

Reply:
so, we dont need if/else here.

Reply:
Done